### ISSUE TITLE
undefined / null / NaN defaults to 0.0

### ISSUE THREAD

**Issue by StephanMeijer** [surveyjs/surveyjs#1563]
### Are you requesting a feature, reporting a bug or asking a question?

bug

### What is the current behavior?

Selects 0 ('No' in screenshot) when value undefined has been seeded

<img width="631" alt="screenshot 2019-02-27 at 15 20 39" src="https://user-images.githubusercontent.com/3169264/53496978-75168080-3aa3-11e9-9574-9ce85491bcdc.png">

### What is the expected behavior?

Should not select anything by default

### How would you reproduce the current behavior (if this is a bug)?

```
    this.model.data = {
      monthly_insurance: null,
      ...this.model.data
    }
```
            {
              title: {
                en: "Do you want to include the micro insurance?",
                ar: "هل تريد الحصول على تأمين الميكروي؟"
              },
              description: {
                en: `The cost of this insurance is JOD 2.2 per month or JOD 1.2 if you do not want to include your children.`,
                ar: "قيمة التأمين 2.5 دينار أو 1.5 دينار شهرياً إذا كنت لا ترغب بشمول أطفالك وزوجك/ زوجتك في التأمين. سيترتب عليك دفعها شهرياً مع كل دفعة من دفعات  تمويلك وحتى سداده بالكامل"
              },
              name: "monthly_insurance",
              type: "radiogroup",
              isRequired: true,
              choices: [
                {value: 2.2, text: {en: "Yes, include my children for 2.2 JOD per month", ar: "نعم أرغب بالحصول على التأمين الميكروي لي ولأطفالي  مقابل 2.5 دينار شهرياً"}},
                {value: 1.2, text: {en: "Yes, but do not include my children, which costs 1.2 JOD per month", ar: "نعم أرغب بالحصول على التأمين المكيروي لي فقط مقابل  1.5 دينار  شهرياً"}},
                {value: 0.0, text: {en: "No", ar: "لا"}},
              ]
            },
```

```

### Specify your

* browser: Chrome 
* browser version: `Version 72.0.3626.119 (Official Build) (64-bit)`
* surveyjs platform (angular or react or jquery or knockout or vue): react
* surveyjs version: `1.0.71`

**Comment by tsv2013** [comment]
Fix will be available in the nearest update, most likely next tuesday

**Comment by StephanMeijer** [comment]
@tsv2013 Any news? :)

**Comment by tsv2013** [comment]
Please check the thread's log:
>tsv2013 closed this in fac57f9 5 days ago

Update had been released last monday
